docker-for-windows/troubleshoot.md

```diff
@@ -286,43 +277,37 @@ script](https://github.com/moby/moby/issues/24388).
 
 ### Virtualization
 
-In order for Docker Desktop for Windows to function properly your machine needs:
+In order for Docker Desktop to function properly your machine needs:
 
 1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview)
    installed and working
 
 2. Virtualization enabled
 
-
 #### Hyper-V
 
-
-Docker Desktop for Windows requires a Hyper-V as well as the Hyper-V Module for Windows
-Powershell to be installed and enabled. The Docker Desktop for Windows installer enables
+Docker Desktop requires a Hyper-V as well as the Hyper-V Module for Windows
```

Comment:
Should this be `Hyper-V` instead of `a Hyper-V`?

Reply:
Fixed in commit 165e8c21.